tareas/clase-6/tarea-1/calculos-para-la-tarea-1.js

```diff
@@ -0,0 +1,29 @@
+function obtenerLaMayorEdad(edadesIntegrantes) {
+    let mayorEdad = edadesIntegrantes[0];
+    for (let i = 0; i < edadesIntegrantes.length; i++) {
+        if (edadesIntegrantes[i] > mayorEdad) {
+            mayorEdad = edadesIntegrantes[i];
+        }
+    }
+    return mayorEdad;
+}
+
+function obtenerLaMenorEdad(edadesIntegrantes) {
+    let edadMinima = edadesIntegrantes[0];
+    for (let i = 0; i < edadesIntegrantes.length; i++) {
+        if (edadesIntegrantes[i] < edadMinima) {
+            edadMinima = edadesIntegrantes[i];
+        }
+    }
+    return edadMinima;
+}
+
+function calcularPromedio(edadesIntegrantes) {
+    let resultado = 0;
+    let promedio = 0;
+    for (let i = 0; i < edadesIntegrantes.length; i++) {
+        resultado = resultado + edadesIntegrantes[i];
+        promedio = resultado / edadesIntegrantes.length;
+    }
+    return promedio;
```

Comment:
una vez que sumaste todo (en resultado), recién por fuera del for simplemente hacés return resultado / edadesIntegrantes.length

Lo que estás haciendo ahora funciona, pero es menos eficiente porque en vez de hacer la división 1 vez, la hacés muchas veces (1 por iteración).